core/src/main/resources/lib/hudson/iconSize_pt_BR.properties

```diff
@@ -21,6 +21,9 @@
 # THE SOFTWARE.
 
 Icon=Ícone
-L=L
-S=S
+
+S=P
+
+L=G
+
```

Comment:
No need to keep a spare line between the entries.

Reply:
ok, I will fix it